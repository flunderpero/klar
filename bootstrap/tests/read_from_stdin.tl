let byte = ""

loop do
    byte = read()
    if byte == "" then
        print("break")
        break
    end
    print(byte)
end
