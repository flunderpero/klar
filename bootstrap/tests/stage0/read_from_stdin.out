s
t
d
i
n
_
t
e
s
t
_
i
n
p
u
t


break
